web_external/layout/layoutHeader.jade

```diff
@@ -79,9 +79,9 @@ nav.navbar.navbar-default.navbar-fixed-top.isic-header-wrapper
                 | Datasets
             if currentUser
               li(role='presentation')
-                a(title='Upload Dataset' href='#dataset/upload')
+                a(title='Contribute Dataset' href='#dataset/create')
                   i.icon-upload
-                  | Upload Dataset
+                  | Contribute Dataset
```

Comment:
I think we're inconsistent in the terminology: "contribute" vs. "create." I'm fine with either here, but maybe it should say "Create Dataset"?